hudi-spark-datasource/hudi-spark3/src/main/java/org/apache/hudi/spark3/internal/ReflectUtil.java

```diff
@@ -43,4 +47,23 @@ public static InsertIntoStatement createInsertInto(boolean isSpark30, LogicalPla
       throw new RuntimeException("Error in create InsertIntoStatement", e);
     }
   }
+
+  public static DateFormatter getDateFormatter(String sparkVersion, ZoneId zoneId) {
+    try {
+      ClassLoader loader = Thread.currentThread().getContextClassLoader();
+      if (sparkVersion.startsWith("3.2")) {
```

Comment:
would define a method `isSpark32()` in `HoodieSparkUtils`

Reply:
good suggestion